shared_web/static/js/pd.js

```diff
@@ -367,6 +368,70 @@ PD.htmlEscape = function (s) {
     return $("<div>").text(s).html();
 };
 
+PD.Filter = {};
+
+PD.Filter.init = function () {
+    // Apply the filter with the initial value of the form
+    PD.Filter.scryfallFilter($("#scryfall-filter-input").val())
+
+    // set up the event handlers for the form
+    $("#scryfall-filter-form").submit(function () {
+        PD.Filter.scryfallFilter($("#scryfall-filter-input").val());
+        return false;
+    });
+    $("#scryfall-filter-reset").click(PD.Filter.reset);
+};
+
+PD.Filter.scryfallFilter = function (query) {
+    if (query == ""){
+        PD.Filter.reset();
+        return;
+    }
+
+    $("#scryfall-filter-submit").attr("disabled", "disabled").text("Loading...");
+    $("#scryfall-filter-reset").attr("disabled", "disabled").text("Loading...");
+    $("#scryfall-filter-form").submit(function () { return false; })
+
+    query = "f:pd and (" + query + ")";
```

Comment:
This will break when viewing /seasons/{not current season}/cards/ – there will be cards that should match the search that do not because they are not legal in the *current* pd season. Scryfall does not offer historical pd season search.